src/app/containers/FrontPageMain/index.jsx

```diff
@@ -32,7 +32,11 @@ const FrontPageMain = ({ frontPageData }) => {
         <GhostWrapper>
           <GridItemConstrainedLarge>
             {groups.map(group => (
```

Comment:
You can get the index from the map :)
```suggestion
             {groups.map((group, index) => (
```